opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -169,6 +174,13 @@ export const batchRoleCapabilities = async (context, user, roleId) => {
   return batchListThroughGetTo(context, user, roleId, RELATION_HAS_CAPABILITY, ENTITY_TYPE_CAPABILITY, { paginate: false });
 };
 
+export const getDefaultHiddenTypes = async (context, userId, userRoles) => {
+  let userDefaultHiddenTypes = [];
+  userRoles.map((role) => role.default_hidden_types?.map((type) => userDefaultHiddenTypes.push(type)));
```

Comment:
Use a map to add something in an array is a bit weird.  Your userDefaultHiddenTypes is just the result of the first map, no need to repush

Reply:
not really because the result of the first map is an array of array... and I want all the types to be in a simple array

Reply:
You can use .flat() function for that